mm2src/coins/utxo/utxo_tests.rs

```diff
@@ -3403,9 +3404,10 @@ fn test_withdraw_p2pk_balance() {
         max: false,
         fee: None,
         memo: None,
+        ibc_source_channel: None,
```

Comment:
nit: we can do `..Default::default()` to avoid writing everything, especially that it might not make sense in the context of the test.

Reply:
Thanks for the suggestion, it's even backward compatible and we will avoid such issues in the future.